cmd/netobserv-ebpf-agent.go

```diff
@@ -39,22 +39,46 @@ func main() {
 
 	logrus.WithField("configuration", fmt.Sprintf("%#v", config)).Debugf("configuration loaded")
 
-	flowsAgent, err := agent.FlowsAgent(&config)
-	if err != nil {
-		logrus.WithError(err).Fatal("can't instantiate NetObserv eBPF Agent")
-	}
+	if config.EnablePano {
```

Comment:
Shouldn't the check be which agent to enable rather than enablePano?

Is there a scenario that we might need both the agents enabled from the userspace?